### DISCUSSION TITLE
Plex shows "some" audiobooks with Multiple icons (one per file), but others as a single book.

### DISCUSSION THREAD

**Discussion by zwavoo** [seanap/Plex-Audiobook-Guide#104] [Q&A] [answered]
I have added a number of audiobooks in the format audiobooks/author/book/*.mp3 and find that regardless of how I set the id3 tags, some books show up with one icon in plex and then tracks or chapters within, or I get each chapter / track as a separate icon on the plex menu. If you can picture that, all it takes is one large multi chapter book, and my screen is full. I would love to know how to group all the files together consistently. Any ideas?

<img width="974" alt="Screenshot 2023-10-02 at 21 00 47" src="https://github.com/seanap/Plex-Audiobook-Guide/assets/30829589/61a8bd64-0ed0-40fd-89ff-dbd532dd0407">

**Comment by seanap** [comment, marked as answer]
What do your filenames look like?  I've always just renamed all the files like `[Title] - pt[Chapter#].mp3` and it seems to work most of the time. Adding the `- pt##` should force plex to treat all the files as 1 media item.  Plex [recommends ](https://support.plex.tv/articles/200265296-adding-music-media-from-folders/) `[Track#] - [Title].mp3`

Plex will also do this to me if the id3 tags for ALBUM are different for each file.

If normalizing the filename and tags still don't work it can be fixed in plex by selecting all of the same covers, click the 3 dots, and choose merge.

**Reply by seanap** [reply]
You may have to do the Plex Dance when trying to fix this after adding the book to plex.  I would move the book folder out of your audiobooks folder, scan library (should remove the book), fix filenames, add the book folder back, scan the library.

**Reply by zwavoo** [reply]
Oh goodness, what an easy solution. I didnt even get to try the filenames as you suggested, but the Merge option is a great fix, Thanks so much!
